lib/postgrex/extensions/binary.ex

```diff
@@ -46,7 +46,10 @@ defmodule Postgrex.Extensions.Binary do
     do: parameters["server_version"] |> Postgrex.Utils.parse_version
 
   def matching(version) when version >= {9, 1, 0},
-    do: [send: "void_send"] ++ matching(0)
+    do: [send: "void_send"] ++ matching({9, 0, 0})
+
+  def matching(version) when version >= {9, 0, 0},
```

Comment:
We don't need a version check here, if someone runs version 8.4 the code will still work.